lib/SPIRVProducerPass.cpp

```diff
@@ -383,6 +383,16 @@ struct SPIRVProducerPassImpl {
       HasNonUniformPointers = true;
     }
   }
+  bool hasConvertToF() { return HasConvertToF; }
+  void setConvertToF() {
+    if (!HasConvertToF && SpvVersion() >= SPIRVVersion::SPIRV_1_4 &&
+        (ExecutionModeRoundingModeRTE(RoundingModeRTE::fp16) ||
+         ExecutionModeRoundingModeRTE(RoundingModeRTE::fp32) ||
+         ExecutionModeRoundingModeRTE(RoundingModeRTE::fp64))) {
+      addCapability(spv::CapabilityRoundingModeRTE);
+      HasConvertToF = true;
```

Comment:
The version could be dropped and the extension could be added if the version is <1.4

Reply:
I don't understand what extension you are talking about when version is <1.4

Reply:
[SPV_KHR_float_controls](http://htmlpreview.github.io/?https://github.com/KhronosGroup/SPIRV-Registry/blob/main/extensions/KHR/SPV_KHR_float_controls.html)

Reply:
Done